client/my-sites/email/inbox/mailbox-selection-list/progress-line-style.scss

```
@@ -0,0 +1,101 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+.progress-line__container {
+	position: fixed;
+	top: 50%;
+	left: 50%;
+	transform: translate( -50%, -50% );
```

Regarding the positioning, I think you should be able to handle this using `display: flex;`. I was able to do that using the following CSS on this element:
```suggestion
	align-content: center;
	display: flex;
	flex-direction: column;
	justify-content: center;
	margin-top: 35vh;
	width: 100%;
```

---

How did I miss **vh**/**vw**? T'was the missing piece. **TIL** 👍 .
BTW the most important bits here are:
```css
display: flex;
flex-direction: column;
margin-top: 35vh;
```

The rest are either defaults or redundant.